test/sharness/t0080-repo.sh

```diff
@@ -219,6 +219,23 @@ test_expect_success "'ipfs refs --unique --recursive (bigger)'" '
 	test_sort_cmp expected actual || test_fsh cat refs_output
 '
 
+get_field_num() {
+  field=$1
+  file=$2
+  num=$(grep "$field" "$file" | awk '{ print $2 }')
+  echo $num
+}
+
+test_expect_success "'ipfs repo stat' succeeds" '
+  ipfs repo stat > repo-stats &&
+  grep "RepoPath" repo-stats &&
+  grep "RepoSize" repo-stats &&
```

Comment:
I would break this up into more fine grained tests, makes debugging easier later on, maybe do:

- ipfs repo stat
- check that it looks right
- add the file and rerun repo stat
- check that the second stat looks right

where each item there is a separate `test_expect_success`